core/lib/url-utils.js

```diff
@@ -4,12 +4,8 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-/**
- * URL shim so we keep our code DRY
- */
-
 import {Util} from '../util.cjs';
-import {LighthouseError} from '../lib/lh-error.js';
+import {LighthouseError} from './lh-error.js';
```

Comment:
lol i didn't make this change, I guess vscode did some house cleaning during the File Rename refactor.